### PR TITLE
docs(main): update plugin config doc

### PR DESCRIPTION
**Pull request by caohuilin**
### <samp>🤖 Generated by Copilot at bbbc1f6</samp>

This pull request improves the plugin config documentation for the `@modern-js/main-doc` package. It edits the English text in `./packages/document/main-doc/docs/en/configure/app/plugins.mdx` and adds a changeset file in `.changeset/hip-grapes-invent.md` to record the patch update.

### <samp>🤖 Generated by Copilot at bbbc1f6</samp>

*  Update plugin config documentation in English and Chinese for `@modern-js/main-doc` package ([link](https://github.com/web-infra-dev/modern.js/pull/3920/files?diff=unified&w=0#diff-edb45e1670a20e6909e1010d9e355c5a295f7b554b024778071257518c1d0924R1-R7), [link](https://github.com/web-infra-dev/modern.js/pull/3920/files?diff=unified&w=0#diff-9851f36409b8723099852b05480d2dc800c7a5c9d04ade54a6edc7dfed691febL12-R44), [link](https://github.com/web-infra-dev/modern.js/pull/3920/files?diff=unified&w=0#diff-9851f36409b8723099852b05480d2dc800c7a5c9d04ade54a6edc7dfed691febL56-R56), [link](https://github.com/web-infra-dev/modern.js/pull/3920/files?diff=unified&w=0#diff-9851f36409b8723099852b05480d2dc800c7a5c9d04ade54a6edc7dfed691febL68-R68),   )

## Checklist

- [x] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/main-doc/docs/en/configure/app/plugins.mdx [hunk 1]
```diff
@@ -9,39 +9,39 @@ sidebar_position: 9
...
-- Use [tools.babel](/configure/app/tools/babel) to configure Babel plugins.
+- To configure Modern.js Builder plugins, use the [builderPlugins](/configure/app/builder-plugins) option.
+- To configure webpack plugins, use the [tools.webpack](/configure/app/tools/webpack) or [tools.webpackChain](/configure/app/tools/webpack-chain) options.
+- To configure Babel plugins, use the [tools.babel](/configure/app/tools/babel) option.
 
-## Plugin type
+## Plugin types
 
-There are three different type of framework plugins built into the Modern.js:
+Modern.js has three types of built-in framework plugins:
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/configure/app/plugins.mdx:24]: Modern.js has three types of built-in framework plugins 
->
Modern.js has three types of plugins

File: packages/document/main-doc/docs/en/configure/app/plugins.mdx [hunk 2]
```diff
@@ -9,39 +9,39 @@ sidebar_position: 9
...
-The ability to customize CLI plugins is currently open Modern.js, and the Server plugin and Runtime plugin will be available in the future.
+Currently, Modern.js has opened up the ability to customize CLI plugins, and Server plugins and Runtime plugins will be opened up later.
 
 ## Plugin execution order
 
-By default, custom plugins are executed sequentially in the order of an array of `plugins`, Modern.js built-in plugins are executed earlier than custom plugins.
+By default, custom plugins are executed in the order of the `plugins` array, and the execution time of built-in Modern.js plugins is earlier than that of custom plugins.
 
-When the plugin uses related fields that control the order, such as `pre` and `post`, the execution order will be adjusted based on the declared fields, see [Relationship between plugins](/guides/topic-detail/framework-plugin/relationship).
+When the plugin uses related fields that control the order, such as `pre` and `post`, the execution order will be adjusted based on the declared fields. Refer to [Relationship between plugins](/guides/topic-detail/framework-plugin/relationship) for more information.
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/configure/app/plugins.mdx:36]: uses related fields -> sets options

File: packages/document/main-doc/docs/en/configure/app/plugins.mdx [hunk 3]
```diff
@@ -9,39 +9,39 @@ sidebar_position: 9
...
 ## Example
 
-Below is an example of using the CLI plugin.
+The following is an example of using CLI plugins.
 
-### Using plugins on npm
+### Use plugins on npm
 
-To use the plugin on npm, you need to install the plugin through the package manager and import it.
+To use plugins on npm, you need to install the plugins through a package manager and import them through `import`.
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/configure/app/plugins.mdx:44]: To use plugins on npm, you need to install the plugins through a package manager and import them through `import`.
->
To use plugins from npm registry, you need to first install the plugins , and import them in `modern.config.ts`  .

File: packages/document/main-doc/docs/en/configure/app/plugins.mdx [hunk 4]
```diff
@@ -53,7 +53,7 @@ export default defineConfig({
 
 ### Use local plugins
 
-Using the plugin in the local code repository, you can import it directly through the relative path import.
+To use plugins from a local code repository, import them directly using a relative path.
```
Comment (xuchaobei) [inline packages/document/main-doc/docs/en/configure/app/plugins.mdx:56]: To use plugins from a local code repository
->

To use local plugins

**Review by xuchaobei — CHANGES_REQUESTED** [review]

**Review by xuchaobei — APPROVED** [review]

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**